Engine/Resource.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace AntEngine
+{
+    /// <summary>
+    /// Represents a resource.
+    /// </summary>
+    public class Resource
+    {
+        public Resource(int a)
+        {
+            Quantity = a;
+        }
+
+        public int Quantity { get; set; }
+    }
```

Comment:
```suggestion
    public class Resource
    {
        public Resource(int a)
        {
            Quantity = a;
        }

        public int Quantity { get; set; }
        public String Name { get; set; }
    }
```

Reply:
> 
> 
> Should a resource have a name String? I think it should.

Ressource extends Entity so it should have a name

Reply:
Resource*

Reply:
:((((